src/test/kotlin/service/SnapshotQueueServiceTest.kt

```diff
@@ -56,6 +56,13 @@ class SnapshotQueueServiceTest : TestBase() {
                 .willReturn(ownerAddress)
         }
 
+        val startBlock = BlockNumber(BigInteger("6"))
```

Comment:
Why is `startBlock = 6`?

Reply:
No real reason, just a random number to use in tests.